README.md

```diff
@@ -88,23 +88,23 @@ Some documented features might not has been published yet, see [change log](./CH
   - `force` options is `true` by default, so it won't run `pre/post` scripts (may change this with `--no-force` flag).
 - `scripts.postupdate` will be executed in host package dir, like while `update` operation.
-- With `--changed` flag yalc will first check if package content has changed before publishing and pushing, it is quick operation, maybe useful for _file watching scenarios_ with pushing on changes.
+- With `--changed` flag yalc will first check if package content has changed before publishing and pushing, it is a quick operation and may be useful for _file watching scenarios_ with pushing on changes.
 
 ### Keep it out of git
 
-- If you are using `yalc'ed` modules temporary while development, first add `.yalc` and `yalc.lock` to `.gitignore`.
-- Use `yalc link`, that won't touch `packages.json`
-- If you use `yalc add` it will change `package.json`, and ads `file:`/`link:` dependencies, if you may want to use `yalc check` in the [precommit hook](https://github.com/typicode/husky) which will check package.json for `yalc'ed` dependencies and exits with error, if you forgot to remove them.
+- If you are using `yalc'ed` modules temporarily during development, first add `.yalc` and `yalc.lock` to `.gitignore`.
+- Use `yalc link`, that won't touch `package.json`
+- If you use `yalc add` it will change `package.json`, and ads `file:`/`link:` dependencies, if you may want to use `yalc check` in the [precommit hook](https://github.com/typicode/husky) which will check package.json for `yalc'ed` dependencies and exits with an error if you forgot to remove them.
 
 ### Keep it in git
 
 - You may want to keep shared `yalc'ed` stuff within the projects you are working on and treat it as a part of the project's codebase. This may really simplify management and usage of shared _work in progress_ packages within your projects and help to make things consistent. So, then just do it, keep `.yalc` folder and `yalc.lock` in git.
 - Replace it with published versions from remote repository when ready.
-- **NB!** - standard non-code files like `README`, `LICENCE` etc. will be included also, so you may want to excluded them in `.gitignore` with a line like `**/.yalc/**/*.md` or you may use `.yalcignore` not to include those files in package content.
+- **NB!** - standard non-code files like `README`, `LICENCE` etc. will be included also, so you may want to exclude them in `.gitignore` with a line like `**/.yalc/**/*.md` or you may use `.yalcignore` not to include those files in package content.
 
 ### Publish/push sub-projects
 
-- Useful for monorepos (projects with multiple sub-projects/packages): `yalc publish package-dir will perform publish operation in nested`package` folder of current working dir.
+- Useful for monorepos (projects with multiple sub-projects/packages): `yalc publish package-dir will perform publish operation in nested `package` folder of current working dir.
```

Comment:
This line is already fixed by #57, so you can undo this change.

Reply:
Have reverted the change!